message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -918,6 +918,84 @@
         <description>Stalling and steering towards the land point</description>
       </entry>
     </enum>
+    <enum name="PLANE_MODE">
+      <description>A mapping of plane flight modes for custom_mode field of heartbeat</description>
+      <entry value="0"  name="PLANE_MODE_MANUAL"/>
+      <entry value="1"  name="PLANE_MODE_CIRCLE"/>
+      <entry value="2"  name="PLANE_MODE_STABILIZE"/>
+      <entry value="3"  name="PLANE_MODE_TRAINING"/>
+      <entry value="4"  name="PLANE_MODE_ACRO"/>
+      <entry value="5"  name="PLANE_MODE_FLY_BY_WIRE_A"/>
```

Comment:
Maybe a little wordy.

Reply:
I'd prefer the clarity/lack of ambiguity of spelling it out here.